odoo/cli/obfuscate.py

```diff
@@ -0,0 +1,230 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import odoo
+import sys
+import optparse
+import logging
+
+from collections import defaultdict
+from psycopg2 import sql
+
+from . import Command
+
+_logger = logging.getLogger(__name__)
+
+class Obfuscate(Command):
+    """Obfuscate datas in a given odoo database"""
```

Comment:
```suggestion
    """Obfuscate data in a given odoo database"""
```
Latin, uncountable :wink: